packages/SwingSet/src/kernel/kernel.js

```diff
@@ -1846,7 +1895,8 @@ export default function buildKernel(
     await null;
     try {
       kernelKeeper.establishCrankSavepoint('start');
-      const { processor, message } = getNextMessageAndProcessor();
+      const { processor, message } =
+        getNextMessageAndProcessor(foreverPolicy());
```

Comment:
With the logic in `getNextMessageAndProcessor`, this change is not needed.
```suggestion
      const { processor, message } = getNextMessageAndProcessor();
```